core/tabs/system-setup/system-cleanup.sh

```diff
@@ -26,16 +26,21 @@ cleanup_system() {
             "$ESCALATION_TOOL" "$PACKAGER" -Rns $(pacman -Qtdq) --noconfirm > /dev/null 2>&1
             ;;
         *)
-            printf "%b\n" "${RED}Unsupported package manager: ""$PACKAGER""${RC}"
-            return 1
+            printf "%b\n" "${RED}Unsupported package manager: ${PACKAGER}. Skipping.${RC}"
             ;;
     esac
 }
 
 common_cleanup() {
-    "$ESCALATION_TOOL" find /var/tmp -type f -atime +5 -delete
-    "$ESCALATION_TOOL" find /tmp -type f -atime +5 -delete
-    "$ESCALATION_TOOL" find /var/log -type f -name "*.log" -exec truncate -s 0 {} \;
+    if [ -d /var/tmp ]; then
+        "$ESCALATION_TOOL" find /var/tmp -type f -atime +5 -delete
+    fi
+    if [ -d /tmp ]; then
+        "$ESCALATION_TOOL" find /tmp -type f -atime +5 -delete
+    fi
+    if [ -d /var/log ]; then
+        "$ESCALATION_TOOL" find /var/log -type f -name "*.log" -exec truncate -s 0 {} \;
+    fi
```

Comment:
Can we do something like this ?

```suggestion
    "$ESCALATION_TOOL" find /var/tmp -type f -atime +5 -delete &> /dev/null || true
    "$ESCALATION_TOOL" find /tmp -type f -atime +5 -delete &> || true
    "$ESCALATION_TOOL" find /var/log -type f -name "*.log" -exec truncate -s 0 {} \; &> || true
```

Reply:
We could but if user doesnt have perms for a file (could happen and will happen with /tmp) it'll fail too

Reply:
The cmd is escalated so there will no perm err